minimal-deployment/minimal-deployment-values.yaml

```diff
@@ -0,0 +1,47 @@
+amalthea:
+  deployCrd: false
```

Comment:
For a functional deployment from nothing in a fresh cluster this should be true.

```suggestion
  deployCrd: true
```